SITE-000 :: update column class names

aclangor opened the pull request:
* Updates the new class name convention from `-*c` to `*-col` for clarity and memorability.
* Updates the changelog

aclangor committed:
SITE-000 :: update class names from `-*c` to `*-col`

aclangor committed:
SITE-000 :: render CSS

aclangor committed:
SITE-000 :: update changelog

aclangor commented:
@ActiveCampaign/site

NickCiliak reviewed (commented):
Good work here. I'm wondering if we need all of the additional classes though. Could we get by with just a few utility classes for centering items on the grid? Otherwise I'm wondering what the application of these might be.

I see them being used to center containers on the middle 10 or middle 8 columns of the grid (that's what I've seen us use so far) so we might be able to get away with 1-col and 2-col offsets. What do you think?

aclangor commented:
@NickCiliak they will primarily be for the instances you mentioned. My initial thought was to limit them as well, but I couldn't come up with a strong argument for limiting them. If you think it is option overload, it's no problem to put more limits in place

NickCiliak commented:
@cameronjbrand I'm in favor of limiting them in order to reduce file size, and if we find ourselves needing more options we can add them in.

aclangor committed:
SITE-000 :: limit the range of classes available

we likely won't need anything above 2-columns of padding or margin, so limit the classes generated to help keep file size down

aclangor committed:
SITE-000 :: render CSS

aclangor committed:
SITE-000 :: update changelog

aclangor commented:
@NickCiliak updated!

NickCiliak reviewed (approved):
🎉
